src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+@import url(https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap);
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  align-items: center;
+  text-transform: uppercase;
+  font-style: normal;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav_list {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+}
+
+.nav_item {
+  display: flex;
+}
+
+.nav_link {
+  font-weight: 500;
+  text-decoration: none;
+  padding: 23px 0;
+  margin-left: 20px;
+  font-size: 12px;
+  color: #000;
```

Comment:
Please pay attention to your css properties order 
https://9elements.com/css-rule-order/